warehouse/ingest-scripts/src/main/resources/bin/ingest/findJars.sh

```diff
@@ -22,8 +22,13 @@ findAccumuloJar (){
 findZookeeperJar(){
   result=$(ls -1 $ZOOKEEPER_HOME/zookeeper-*.jar 2>/dev/null | head -1)
   [[ -f $result ]] || result=$(ls -1 $ZOOKEEPER_HOME/lib/zookeeper-*.jar | head -1)
+  echo $result
+}
+findZookeeperJuteJar(){
```

Comment:
This new method only differs from the one above by the name of the jat its looking for. Should we instead pass in a parameter and use one function

Reply:
Will do!